README.md

```diff
@@ -2,4 +2,99 @@
 Default implementation of Ruuvi Bluetooth library for Android
 
 # This link is very useful:
-https://github.com/ruuvi/ruuvi-sensor-protocols
\ No newline at end of file
+https://github.com/ruuvi/ruuvi-sensor-protocols
+
+# Usage example
+
+Add the JitPack repository to your projects build.gradle file
+```gradle
+allprojects {
+    repositories {
+        maven { url "https://www.jitpack.io" }
+        ...
+    }
+}
+```
+
+Add the library and its interface to your project by adding following as dependencies in app/build.gradle
+```gradle
+dependencies {
+    implementation 'com.github.ruuvi:com.ruuvi.bluetooth:2cedd919b2'
+    implementation 'com.github.ruuvi:com.ruuvi.bluetooth.default:48cf870335'
```

Comment:
Let's use 31e717bcb7 version. I fixed few bugs there

Reply:
Is there a way to refer to latest release? e.g. com.github.ruuvi:com.ruuvi.bluetooth:1.0.0^
and then make releases on GitHub or something else?

Reply:
> Is there a way to refer to latest release? e.g. com.github.ruuvi:com.ruuvi.bluetooth:1.0.0^
> and then make releases on GitHub or something else?

This makes most sense I think. Should we release the current master as 1.0.0? @andreevdenis

Reply:
I think we can wait till 1.2.12 will be released and we we move 31e717b version of lib to master as 1.0.0.